### DISCUSSION TITLE
New Demo: Collapsible Minimap

### DISCUSSION THREAD

**Discussion by kumilingus** [clientIO/joint#2338] [News]
While the **diagram minimap** is a great tool to help you navigate through large diagrams, it is not a tool that users are expected to use most of the time. To reduce clutter in the UI, the user can be allowed **to show and hide the navigator** at their will. And how to do this specifically with _JointJS+_ is the subject of [today's demo](https://codepen.io/jointjs/full/LYMLRGj).

<img height="290" alt="image" src="https://github.com/clientIO/joint/assets/3967880/83881513-6bca-4a39-9f0b-a3182a51cd98">
<img height="290" alt="image" src="https://github.com/clientIO/joint/assets/3967880/ff0d973a-ea50-4922-90d4-39635fd539b6">

**Comment by pandika-csgi** [comment]
I've downloaded jointplus v4.0 but inside KitchenSink Angular16 there's no code about collapsible minimap like the demo did here: https://www.jointjs.com/demos/kitchen-sink-app

is there something I'm missing? @kumilingus 
Thank you

**Reply by kumilingus** [reply]
The code for the collapsible minimap can be found here: https://www.jointjs.com/demos/collapsible-minimap

The Kitchen Sink redesign has not been released yet.
